### DISCUSSION TITLE
Disable animations / page transitions

### DISCUSSION THREAD

**Discussion by taenzerme** [Laravel-Backpack/community-forum#635] [Show and Tell]
Hello all,

is there an easy way to disable all transitions / animations (especially on page load)?

Best,
Sebastian

**Comment by taenzerme** [comment]
To answer my own question, a simple quick way to do this would be using a custom css file:

```
* {
    animation-duration: 0s !important;
    animation-delay: 0s !important;
    transition-duration: 0s !important;
    transition-delay: 0s !important;
}
```

Is there another way to remove animate.css completely?

**Comment by pxpm** [comment]
Hey @taenzerme either that or create an overwrite of the file https://github.com/Laravel-Backpack/CRUD/blob/c39ccb8074e15b075b8f2e09275d5514196955f0/src/resources/views/ui/inc/styles.blade.php#L1

If you place it in `resources/views/vendor/backpack/ui/inc/styles.blade.php` Backpack should pick that file instead of the package one. 

Although notice that by overwriting this file you will not get updates we do on it, so you should take care to keep it up-to-date with the core. (that file shouldn't change much, but the possibility exists). 

Apart from your suggestion and my own, I am not seeing at the moment other way to disable the animations.

Cheers

**Reply by taenzerme** [reply]
Perfect, that helps! Thanks!
